docs/getting-started/server/guide.md

```diff
@@ -22,6 +22,13 @@ Before you start: make sure you’ve installed the recommended
 
 :::
 
+:::info
+
+A guide for developing in VS Code Dev containers can be found [here](./guide-vscode) if you prefer
```

Comment:
⛏️ Capital "C" for "Containers".

This is more of my style I suppose, but I try to future-proof this a bit with something like a list format. This could instead be reworded as a collection of guides available, and then you might want to have this in a subfolder of "guides". We have VS Code and VS material here and perhaps they should follow your pattern here so it isn't unique and on its own.

I realize that's easy to ask, but it'd be a nice structure change.